packages/viem/src/index.ts

```diff
@@ -97,21 +95,9 @@ export async function createAccount(input: {
     // override provided `ethereumAddress`
     ethereumAddress = signWith;
   } else if (!ethereumAddress) {
-    // we have a private key ID, but not an ethereumAddress
-    const data = await client.getPrivateKey({
-      privateKeyId: signWith,
-      organizationId: organizationId,
+    throw new TurnkeyActivityError({
```

Comment:
Or does this not make sense because you cant do getPrivateKeyAccount?

Reply:
That's right; this path is the non-default path, for people who "know what they're doing". Specifically the people that know exactly the private key/address they want to sign with. So the main difference between this and `createAccount` is that this (`createAccountWithAddress`) is intended to avoid making that query